js/modules/k6/metrics/metrics.go

```diff
@@ -129,9 +129,10 @@ func (m Metric) add(v goja.Value, addTags ...map[string]string) (bool, error) {
 	}
 
 	tags := state.CloneTags()
-	for _, ts := range addTags {
```

Comment:
I guess this technically is a brekaing change as we previouls supported multiple maps .... but that seems like a bad idea to begin with - just maybe we should put in the release notes.

Reply:
I updated the PR description